core/controllers/base_test.py

```diff
@@ -377,8 +377,7 @@ def test_renders_error_page_with_iframed(self):
             '/mock_iframed', expected_status_int=500)
 
         self.assertIn(
-            'Uh-oh! The Oppia exploration you requested may have been removed '
-            'or deleted.', response.body)
+            '<oppia-error-iframed-page-root></oppia-error-iframed-page-root>', response.body)
```

Comment:
Where does this change come from?

Reply:
This change comes from `error-iframed.mainpage.html` file.
Basically, I have removed angularjs from this page. The angularjs allowed trasclusion in root components but angular doesnot allow it. So, I had to create a wrapper component i.e. `oppia-error-iframed-page-root`.

Reply:
Thanks, that helps!